contracts/exchange-libs/contracts/src/LibOrder.sol

```diff
@@ -133,6 +133,12 @@ library LibOrder {
         // ));
 
         assembly {
+            // Assert order offset (this is an internal error that should never be triggered)
+            let orderOffset := order
+            if lt(orderOffset, 32) {
+                revert(0, 0)
```

Comment:
It's probably more correct to use `invalid()` here since this should never be triggered. I don't have a strong preference either way.

Reply:
Neat, didn't know about that!

Reply:
me too :astonished: